build/index.html

```diff
@@ -26,13 +26,14 @@
+          <img src="assets/smallcimcwhite.svg" alt="CIMC">
```

Comment:
we can probably inline the svg so we wouldnt need a new svg file for every color. its mostly duplicate info cept for the color part. with react we can also create a react component that takes a color property. but i guess we can optimize this later.